### PR TITLE
Adjust dark mode colors

### PR DESCRIPTION
**Pull request by mattahorton**
| dark | light |
| ---- | ---- |
| CleanShot 2022-09-26 at 18 15 46@2x | CleanShot 2022-09-26 at 18 16 06@2x |

This is not very clean code at all, but it does do the job to address #14. If you all want, I'm happy to break this isn't pieces and pass each argument only the necessary options.

It would be nice if Gooey let us declare some of this at the global level and pass it down like CSS, but I don't think that's an option.

### CODE REVIEW

**Review by harrislapiroff — CHANGES_REQUESTED** [review]
Thanks so much @mattahorton! This looks pretty great! One change request from me:

Since there isn't actually overlap between the arguments fed to the `Gooey` decorator and the `gooey_options` dict, I don't think we should reuse the same dict—it could cause confusion when reading the code. Can you put the new field options out into a separate variable, maybe called `FIELD_COLORS_DARK`.

Then in each field you could use `gooey_options=FIELD_COLORS_DARK if IS_DARK else {}`

File: stagewhisper/__main__.py [hunk 1]
```diff
@@ -39,33 +43,33 @@ def cli():
     better hints to Gooey for how to render widgets
     """
     parser = GooeyParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    basic_group = parser.add_argument_group("Basic")
-    advanced_group = parser.add_argument_group("Advanced Options")
+    basic_group = parser.add_argument_group("Basic", gooey_options=APPEARANCE_DARK if IS_DARK else APPEARANCE_LIGHT)
```
Comment (crazy4pi314) [inline stagewhisper/__main__.py:46]: An example of what I think @harrislapiroff suggested:
```suggestion
    basic_group = parser.add_argument_group("Basic", gooey_options=FIELD_COLORS_DARK if IS_DARK else {})
```

**Review by crazy4pi314 — DISMISSED** [review]
Second @mattahorton, made a code suggestion that I think reflects the option passing to gooey. TYSM!

**Review by harrislapiroff — APPROVED** [review]
Looks great!

**Comment by crazy4pi314** [comment]
🎉 🎉 🎉

**Pull request merged**